e2e/libs/keywords/recurring_job_keywords.py

```diff
@@ -17,15 +16,15 @@ def create_snapshot_recurring_job_for_volume(self, volume_name):
         self.recurring_job.create(job_name, task="snapshot")
         self.recurring_job.add_to_volume(job_name, volume_name)
         self.recurring_job.get(job_name)
-        logging.warn(f'==> create recurring job {job_name} for volume {volume_name}')
+        logging(f'Created recurring job {job_name} for volume {volume_name}')
```

Comment:
Can do below to have logs around the call functions, especially for exported functions. You can review others to have the same pattern.  

```python
logging("Creating ....")
....
logging("Created ...")
```

Again, this can be automatically implemented via metaclass such as https://stackoverflow.com/questions/16961914/python-logging-all-of-a-class-methods-without-decorating-each-one, so you don't need to manually add a log for each function/method. This is a beauty of dynamic lang or lang supporting metaclass or macro. You can improve this later.

Reply:
Created https://github.com/longhorn/longhorn/issues/6758 for this.